### PR TITLE
create 2 databases to separate DATA and CODE

### PR DESCRIPTION
**Pull request by SylvainGuilbaud**
update the README.md for unit testing

### CODE REVIEW

**Comment by evshvarov** [comment]
@SylvainGuilbaud , thanks a lot for the PR! Is it so important to separate CODE and DATA databases? Is it a common approach? What are the benefits?

File: src/dc/sample/ObjectScript.cls [hunk 1]
```diff
@@ -2,12 +2,43 @@ Class dc.sample.ObjectScript
...
     return a
 }
 
+ClassMethod Python() As %Status [ Language = python ]
+{
+import iris    
+from datetime import datetime
+a=42 
+print("Hello World! from",iris.cls('dc.sample.ObjectScript')._ClassName(1),"in Python language")
+print("This is InterSystems IRIS with version",iris.cls('%SYSTEM.Version').GetVersion())
```
Comment (daimor) [inline src/dc/sample/ObjectScript.cls:38]: 'dc.sample.ObjectScript' can be replaced by `__name__`

`iris.cls('%SYSTEM.Version').GetVersion()` = `iris.system.Version.GetVersion()`

File: src/dc/sample/ObjectScript.cls [hunk 2]
```diff
@@ -2,12 +2,43 @@ Class dc.sample.ObjectScript
...
+{
+    return:python ..Python()
+    return:'python ..ObjectScript()
+}
+
+ClassMethod ObjectScript() As %Status
 {
     set a=42 
-    write "Hello World!",!
+    write "Hello World! from ",$CLASSMETHOD("dc.sample.ObjectScript","%ClassName",1)," in ObjectScript language",!
```
Comment (daimor) [inline src/dc/sample/ObjectScript.cls:26]: why not just `..%ClassName(1)`?

Comment (SylvainGuilbaud) [reply src/dc/sample/ObjectScript.cls:26]: It is just to illustrate the $CLASSMETHOD use + offer a parallel with python syntax.

File: src/dc/sample/ObjectScript.cls (hunk 1 above)
Comment (SylvainGuilbaud) [reply src/dc/sample/ObjectScript.cls:38]: Thanks !!

**Comment by SylvainGuilbaud** [comment]
> @SylvainGuilbaud , thanks a lot for the PR! Is it so important to separate CODE and DATA databases? Is it a common approach? What are the benefits?

Storing DATA and CODE in 2 different databases offers many advantages in terms of security, performances, administration, testing, etc.
i.e: several namespaces can share the same CODE, without sharing their DATA (an easy way to provide multi-tenant on a single instance) ; you can in the reverse test same DATA with separate versions of CODE ; you can mount CODE in ReadOnly ; etc.

**Comment by eduard93** [comment]
Good idea.

**Pull request merged**